styx-scheduler-service/src/test/java/com/spotify/styx/test/FlyteAdminClientRunnerTest.java

```diff
@@ -117,4 +135,92 @@ public void testThrowsCreateExecutionExceptionForUnknownException() {
         FlyteRunner.CreateExecutionException.class,
         () -> flyteRunner.createExecution("exec", FLYTE_EXEC_CONF));
   }
-}
+
+  @Test
+  public void testPollFlyteExecutionIdCannotBeNull() {
+    assertThrows(
+        NullPointerException.class,
+        () ->    flyteRunner.poll(null, RunState.create(null, null))
+    );
+  }
+
+  @Test
+  public void testPollRunStateCannotBeNull() {
+    assertThrows(
+        NullPointerException.class,
+        () ->    flyteRunner.poll(FlyteExecutionId.create("flyte-test", "testing", "test-run-state-cannot-be-null"), null)
+    );
+  }
+
+  @Test
+  public void testTransititionRunningToTerminateSuccessfulRun() throws Exception {
+    Workflow workflow = Workflow.create("id", configuration());
+    WorkflowInstance workflowInstance = WorkflowInstance.create(workflow.id(), "2016-03-14");
```

Comment:
These could be in constants as they are reused in several tests

Reply:
I think it's ok since the moment we will need to change the configuration or parametrise something, we will have to revert to what we see here.
How strongly do you feel about the refactoring?

Reply:
Not super, but it looked that you needed to parameterized the response to the response of `getExecution()` the `workflow` and `workflowInstance` looks the same on both tests